style.css

```diff
@@ -74,6 +74,110 @@ footer a {
   color: #ebebff;
 }
 
+.modal {
+  display: none;
+  position: fixed;
+  justify-content: center;
+  align-items: center;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #191765;
+}
+
+.modal-content img {
+  width: 100%;
+  height: 50%;
+  margin-top: 20px;
+}
+
+.work-nav-1 img {
+  height: 18px;
+  width: 21px;
+}
+
+.modal-content {
+  background-color: rgba(37, 47, 137, 0.08);
```

Comment:
- [ ] It may be a good idea to fix the modal-content's `background-color` because It is not the same as the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10). 😉
- I think, you need to change the opacity of the `background-color` like `background-color: rgba(37, 47, 137, 0.8)`